ui/package.json

```diff
@@ -34,7 +34,7 @@
     "@splunk/themes": "^0.18.0",
     "@splunk/ui-utils": "^1.6.0",
     "@splunk/visualization-context": "^26.0.0",
-    "@storybook/test": "^8.0.10",
```

Comment:
I am not sure if updating packages as a good idea for `docs` PR

Reply:
maybe true i will revert it. added them because it is update only of storybook so potentially only docs packages, but yea looking at last issue good point to not update it like that

Reply:
reverted